jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/Server.java

```diff
@@ -539,12 +539,12 @@ protected void doStart() throws Exception
             if (getStopAtShutdown())
                 ShutdownThread.register(this);
 
-            //Register the Server with the handler thread for receiving
-            //remote stop commands
-            ShutdownMonitor.register(this);
-
-            //Start a thread waiting to receive "stop" commands.
-            ShutdownMonitor.getInstance().start(); // initialize
+            ShutdownService shutdown = getShutdown();
```

Comment:
Don't have a single `getShutdown()` method.   A server might participate in multiple shutdown services: the deprecated static one and more than one configured as a bean one.    So explicitly call the static (if it is configured), then each bean of the matching type.